core-web/libs/sdk/react/src/lib/components/Column/Column.tsx

```diff
@@ -35,12 +37,13 @@ export function Column({ column }: ColumnProps) {
 
     const combinedClasses = combineClasses([styles[endClass], styles[startClass]]);
 
-    const columnProps = isInsideEditor
-        ? {
-              'data-dot': 'column',
-              'data-testid': 'column'
-          }
-        : {};
+    const columnProps =
+        UVEState?.mode === UVE_MODE.EDIT
```

Comment:
2 files... 3 times this coparison... we should do something about it.

Reply:
Why some cases is

```
getUVEState()?.mode !== UVE_MODE.EDIT
```

And other is like this?

Reply:
That's why I asked to export a util because we also need this internally

Reply:
This changes were rollback, so we can discuss this when we get there